test/server/cards/03_TWI/leaders/WatTamborTechnoUnionForeman.spec.ts

```diff
@@ -0,0 +1,128 @@
+describe('Wat Tambor, Techno Union Foreman', function () {
+    integration(function (contextRef) {
+        describe('Wat Tambor\'s leader ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['battlefield-marine', 'guardian-of-the-whills'],
+                        spaceArena: ['green-squadron-awing'],
+                        leader: 'wat-tambor#techno-union-foreman',
+                        resources: 4,
+                    },
+                    player2: {
+                        groundArena: ['admiral-yularen#advising-caution'],
+                    },
+                });
+            });
+
+            it('should not give +2/+2 to a unit if no friendly unit was defeat this phase', function () {
```

Comment:
Same comment as Asajj, let's put this test after the below one using `moveToNextActionPhase()`